.clang-format

```diff
@@ -5,6 +5,7 @@ AlignEscapedNewlines: Right
 AllowAllParametersOfDeclarationOnNextLine: false
 BinPackParameters: false
 BreakBeforeBraces: Custom
+SortIncludes: false
```

Comment:
Why do we need this change? In general I do not like these type of changes as it indicates that headers should be in a specific order and any inadvertent change will introduce issues.

Reply:
Removed.